Overview.src.html

```diff
@@ -160,6 +160,28 @@ <h2 class="short">Conformance</h2>
 
 <hr>
 
+<p>
+The technique of Token Binding allows user agents to work with servers to protect credentials
+like HTTP cookies.
+A <dfn title=concept-token-binding>Token Binding</dfn> is established by the user agent generating
+a public-private key-pair for the server and proving possession of the private key on every request
+to the server, as described in
+<a href="https://tools.ietf.org/id/draft-ietf-tokbind-protocol-10.xml#rfc.section.1">section 1</a>
+of the Token Binding Protocol spec.
+A <dfn title=concept-token-binding-id>Token Binding Id</dfn> is the representation of a
+<span title=concept-token-binding>Token Binding</span>, as described in
+<a href="https://tools.ietf.org/id/draft-ietf-tokbind-protocol-10.xml#rfc.section.3.2">section 3.2</a>
+of the Token Binding Protocol spec.
+
+A <dfn title=concept-token-binding-message>Token Binding Message</dfn> is transmitted by
+a user agent to a server to prove possession of
+<span title=concept-token-binding>Token Binding</span> keys.
+The value of a <span title=concept-token-binding-message>Token Binding Message</span> is specified
+in <a href="https://tools.ietf.org/id/draft-ietf-tokbind-protocol-10.xml#rfc.section.3">section 3</a>
+of the Token Binding Protocol spec.
```

Comment:
The way I was thinking of this was that we could add a subsection to Infrastructure, just like "Client hints list" is a subsection.